packages/core/src/logger/logger.interface.ts

```diff
@@ -9,6 +9,7 @@ export type LoggerOptions = {
   type: string;
   message: string;
   level?: LoggerLevel;
+  data?: unknown;
```

Comment:
Wouldn't it be better to specify the interface as a `Record<string, unknown>`? To be sure that it is an object with key-value pairs. Eq. with already defined `payload` attribute.

Reply:
Yeah, it's a good idea